Gruntfile.js

```diff
@@ -278,6 +278,10 @@ module.exports = function(grunt) {
       translations: {
         files: ['translations/*'],
         tasks: ['appcache', 'deploy']
+      },
+      node_modules: {
+        files: ['node_modules/**/*'],
```

Comment:
As long as nothing is pulling in non-js files with `require()`.  I know for starters that enketo pulls in at least two `.json` files.  We override both of those in our browserify config, so it wouldn't actually be a problem in that case, but:

@garethbowen @mandric I haven't measured, but I think the performance implications of watching extra files & dirs is probably negligible considering how long it actually takes to (1) re-browserify the entire JS stack, including node-modules, and (2) re-kanso push.  On my machine, **1** can often take 15+ seconds, and **2** can take 30.

So on balance, I think it's safer to stick with `node_modules/**/*`, as the performance boost from specifying `*.js` is likely negligible, and there's potential for weird behaviour that would be very hard to diagnose.